js/interface.js

```diff
@@ -2076,8 +2076,14 @@ function select2FADevice(data) {
 }
 
 function prompt2FA() {
+  var region = Fliplet.Env.get('user').auth_token.substr(0, 2);
+  var serverLocations = {
+    eu: 'Dublin, Ireland',
+    us: 'San Francisco, US'
+  };
+
   return Fliplet.Modal.prompt({
-    title: 'Apple has sent a verification code to your devices. Please enter the code to continue.<div class="alert alert-info alert-sm"><strong>Note:</strong> If you don\'t receive the code, please check your Apple account settings.</div>',
+    title: 'Apple has sent a verification code to your devices from <strong>' + serverLocations[region] + '</strong>. Please enter the code to continue.<div class="alert alert-info alert-sm"><strong>Note:</strong> If you don\'t receive the code, please check your Apple account settings.</div>',
```

Comment:
`(serverLocations[region] || serverLocations.eu)` so it doesn't break when we add a new region